src/main/resources/static/app/request/request-review.html

```diff
@@ -1,4 +1,17 @@
 <div>
+  <h4 class="h4 h4-palga">{{'Request type'|translate}}</h4>
+  <hr>
+
+  <div class="row">
+    <div class="col-sm-6 form-control-static" >
+      <select class="form-control" name="requestType" id="requestType"
+              ng-model="request.requestType" ng-options="type | translate for type in requestTypes"
+              ng-required="true">
+        <option value="" ng-selected="selected">&ndash; {{'Select request type' | translate}} &ndash;</option>
```

Comment:
Apparently not clearly mentioned in the ticket: when selecting 'Other', the user should be able to type a value (from the proposal document: 
> Overig → er opent een tekst veld waar  we zelf wat kunnen in typen

Reply:
That custom value could be saved in the same property. You could check the specialism (in user registration) for an example.

Reply:
it's updated now